.travis.yml

```diff
@@ -9,6 +9,9 @@ services:
 env:
   - TOX_ENV=py27
 
+git:
+  submodules: false
+
```

Comment:
Correct. It did not work until it was done explicitly, see https://travis-ci.org/afgane/docker-galaxy-stable/builds/92776644.